CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
+
+NOTE: This file is no longer being updated. For release notes follow [GitHub releases](https://github.com/lsd-rs/lsd/releases).
```

Comment:
by saying GitHub auto-generates release notes, I mean https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes

We don't have to update this file and will generate by PRs when releasing.

Reply:
Not sure what you would like me to do here. I understand what you are trying to achieve, but not what is the future of this CHANGELOG file. What should the notice say? If there should be a notice at all?